examples/helm/helm_gke.sh

```diff
@@ -28,13 +28,8 @@ echo "=> install pomerium with helm"
 helm install \
 	pomerium \
 	pomerium/pomerium \
-	--set proxy.service.type="NodePort" \
-	--set authenticate.service.type="NodePort" \
-	--set config.sharedSecret=$(head -c32 /dev/urandom | base64) \
-	--set config.cookieSecret=$(head -c32 /dev/urandom | base64) \
-	--set ingress.secret.name="pomerium-tls" \
-	--set ingress.secret.cert=$(base64 -i "$HOME/.acme.sh/*.corp.beyondperimeter.com_ecc/fullchain.cer") \
-	--set ingress.secret.key=$(base64 -i "$HOME/.acme.sh/*.corp.beyondperimeter.com_ecc/*.corp.beyondperimeter.com.key") \
+	--set ingress.secret.cert="$(base64 -i $HOME/.acme.sh/*.corp.beyondperimeter.com_ecc/fullchain.cer)" \
```

Comment:
Does this script still work with these updates?

Reply:
Yes. My understanding is that the values removed from this command are either the default provided by our helm file, or set in `values.yaml`.

The idea here was to set as many things as possible in files that could be version-controlled and set up a IaC.